pkg/diagnostics/cluster/rolebindings.go

```diff
@@ -80,8 +81,9 @@ func (d *ClusterRoleBindings) Check() types.DiagnosticResult {
 		if err != nil {
 			r.Error("CRBD1002", err, fmt.Sprintf("Unable to get clusterrolebinding/%s: %v", changedClusterRoleBinding.Name, err))
 		}
+		actualRBACClusterRole, err := util.ClusterRoleBindingToRBAC(actualClusterRole)
```

Comment:
Check the error.

Reply:
Added error check

Is it OK to introduce a new error id?

Reply:
Absolutely. The only purpose of an error ID is to be able to identify the problem uniquely in automation even if the text changes. So just make any new ones unique.